packages/config/src/log/cleanup.js

```diff
@@ -6,14 +6,24 @@ const { removeEmptyObject, removeEmptyArray } = require('./remove')
 
 // Make sure we are not printing secret values. Use an allow list.
 const cleanupConfig = function({
-  build: { base, command, commandOrigin, environment = {}, functions, ignore, processing, publish } = {},
+  build: { base, command, commandOrigin, environment = {}, functions, edge_handlers, ignore, processing, publish } = {},
```

Comment:
Unrelated to this PR, but should we open an issue to change `commandOrigin` to `command_origin`?
Also, can we also return `edge_handlers_output_dir` (or something similar) to be used by the CLI?
Context https://github.com/netlify/cli/pull/1244#discussion_r490554445

Reply:
For `command_origin`: https://github.com/netlify/build/issues/1834

For `edge_handlers_output_dir`, neither `@netlify/config` nor `@netlify/build` knows nor uses that constant. This is defined and used internally by the Edge handlers plugin at the moment. When it comes to making that plugin share the constant to Netlify CLI, I think this should be tracked in https://github.com/netlify/build/issues/1822#issuecomment-695772337.